packages/typespec-python/test/azure/generated/payload-pageable/payload/pageable/aio/operations/_operations.py

```diff
@@ -180,43 +178,30 @@ def request_query_response_body(
         }
         error_map.update(kwargs.pop("error_map", {}) or {})
 
-        def prepare_request(next_link=None):
-            if not next_link:
-
-                _request = build_server_driven_pagination_continuation_token_request_query_response_body_request(
-                    token=token,
-                    foo=foo,
-                    bar=bar,
-                    headers=_headers,
-                    params=_params,
-                )
-                path_format_arguments = {
-                    "endpoint": self._serialize.url(
-                        "self._config.endpoint", self._config.endpoint, "str", skip_quote=True
-                    ),
-                }
-                _request.url = self._client.format_url(_request.url, **path_format_arguments)
-
-            else:
-                _request = HttpRequest("GET", next_link)
-                path_format_arguments = {
-                    "endpoint": self._serialize.url(
-                        "self._config.endpoint", self._config.endpoint, "str", skip_quote=True
-                    ),
-                }
-                _request.url = self._client.format_url(_request.url, **path_format_arguments)
+        def prepare_request(_continuation_token=None):
```

Comment:
then the underscore could be removed?

Reply:
I still hope to keep it to avoid potential naming conflict.